### DISCUSSION TITLE
Scratch Addons v10.0 vs v1.22.0

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#3779] [Discussion]
(I first found out about Scratch Addons around v1.15)

I wanted to see what Scratch Addons 1.0 was like, so I download `scratchaddons-v1.0.0.zip` and loaded it up into Edge and it worked. The first thing I noticed is that the Scratch Addons pop-up has _completely_ changed!

![popup](https://user-images.githubusercontent.com/81489795/141375968-ba1f53ae-9139-4238-8a7c-c3a3116d1977.png)

The setting page has also had a lot of changes, but nothing like the popup:

![image](https://user-images.githubusercontent.com/81489795/141376604-4dbb2234-30ae-4049-bb36-3e603c30bb78.png)

Some addons like `discuss-button` are still almost exactly the same as in v1.0
![image](https://user-images.githubusercontent.com/81489795/141377517-7926f873-132c-4365-9d5b-d8ace644d465.png)

Others, like `editor-dark mode` have completely changed:
![Untitled](https://user-images.githubusercontent.com/81489795/141377914-6208888c-3194-4504-b8a2-c6e4fd1621c4.png)

**Comment by TheColaber** [comment]
Tis been a very long way....

**Reply by WorldLanguages** [reply]
@Samq64 By the way don't enable "more links" on any version lower than 1.4 lol

**Reply by WorldLanguages** [reply]
https://github.com/ScratchAddons/ScratchAddons/security/advisories/GHSA-6qfq-px3r-xj4p

**Comment by Cookiesmuch** [comment]
Ah yes... Beautiful. Though I have not been there for scratch addons debut, I do acknowledge the growth of its code, and its community. as Griffpatch once said; "a budding group of software engineers".

**Reply by WorldLanguages** [reply]
Where did he say that? lol

**Reply by TheColaber** [reply]
@WorldLanguages I see you've never played getting over it

**Comment by cobaltt7** [comment]
The popup was taken pretty much directly from Scratch Messaging, while everything else was made from scratch.

**Reply by Samq64** [reply]
That makes sense.

**Comment by Secret-chest** [comment]
I have started to use it on 1.3 release

**Reply by cobaltt7** [reply]
https://github.com/ScratchAddons/ScratchAddons/security/advisories/GHSA-6qfq-px3r-xj4p

Don't enable more links

**Comment by Mbrick2** [comment]
Wheres the file. cant find in branches

**Reply by CST1229** [reply]
File for what?

**Reply by Samq64** [reply]
`scratchaddons-v1.0.0.zip` from the releases page.
